storage/README.md

```diff
@@ -28,6 +29,41 @@ amount of data at any time.
 
 ## Samples
 
+### ACL (Access Control Lists)
+
+View the [documentation][acl_docs] or the [source code][acl_code].
+
+__Usage:__ `node acl --help`
+
+```
+Commands:
```

Comment:
Generated in the "I don't have to even paste things in, I just run a script" sense.